src/pymor/parameters/base.py

```diff
@@ -572,6 +572,57 @@ def sample_randomly(self, count=None):
         else:
             return [get_param() for _ in range(count)]
 
+    def sample_logarithmic_uniformly(self, counts, base):
+        """Logarithmically uniform sample |parameter values| from the space.
+
+        Parameters
+        ----------
+        counts
+            Number of samples to take per parameter and component
+            of the parameter. Either a dict of counts per |Parameter|
+            or a single count that is taken for each parameter in |Parameters|.
+
+        Returns
+        -------
+        List of |parameter value| dicts.
+        """
+        if isinstance(counts, dict):
+            pass
+        else:
+            counts = {k: counts for k in self.parameters}
```

Comment:
```suggestion
        if not isinstance(counts, dict):
            counts = {k: counts for k in self.parameters}
```

Reply:
Oh, actually I have made use of already adapted sample_uniformly and sample_randomly when I was writing the code in order to make sure that I comply everything. Thanks for the suggestion I commit them right away.